lib/msal-browser/docs/iframe-usage.md

```diff
@@ -0,0 +1,102 @@
+# Using iframes with MSAL
+
+MSAL.js can be used with iframed applications under restricted conditions:
+
+* You **cannot** iframe the Azure AD login UX itself. Specifically, the service will refuse to render prompts for credential entry and consent by throwing the `X-FRAME OPTIONS SET TO DENY` error. This restriction is due to prevent [clickjacking attacks](https://owasp.org/www-community/attacks/Clickjacking).
```

Comment:
B2C service does allow this: https://docs.microsoft.com/en-us/azure/active-directory-b2c/embedded-login?pivots=b2c-custom-policy

Since MSAL prevents redirects in iframe, apps will need to setup `system.allowRedirectIframe` to `true` to enable this feature.

Reply:
gotcha -added a remark